lib/datasource/github-releases/index.ts

```diff
@@ -35,13 +37,17 @@ export async function getReleases({
   if (cachedResult) {
     return cachedResult;
   }
-  const url = `https://api.github.com/repos/${repo}/releases?per_page=100`;
+  const sourceUrlBase = depHost ?? `https://github.com/`;
```

Comment:
Can you separate this into a new PR? I'm surprised we didn't have a need for enterprise github releases before.